doc/index.rst

```diff
@@ -2,6 +2,225 @@
  Welcome to Ceph
 =================
 
+Ceph is a storage platform. 
+
+Ceph makes possible object storage, block storage, and file storage. It can be used to build cloud infrastructure and web-scale object storage. 
+
+The procedure on this page explains how to set up a three-node Ceph cluster, the most basic of setups.  
+
+Basic Three-Node Installation Procedure
+=======================================
+.. highlight:: console 
+
+
+Installing the First Node
+-------------------------
+
+#. Install a recent, supported Linux distribution on a computer.
+#. Install docker. On Fedora or Centos::
+
+   $ sudo dnf install docker 
+
+  on Ubuntu or Debian::
+
+   $ sudo apt install docker.io
+
+#. Fetch the cephadm utility from github to the computer that will be the Ceph manager::
+
+   $ curl --silent --remote-name --location https://github.com/ceph/ceph/raw/master/src/cephadm/cephadm
+
+#. Make the cephadm utility executable::
+
+   $ sudo chmod +x cephadm
+
+#. Find the ip address of the node that will become the first Ceph monitor::
+
+   $ ip addr
+
+#. Using the ip address that you discovered in the step immediately prior to this step, run the following command::
+
+   $ sudo ./cephadm bootstrap --mon-ip 192.168.1.101 --output-config ceph.conf --output-keyring ceph.keyring --output-pub-ssh-key ceph.pub 
+   
+  The output of a successful execution of this command is shown here::
+
+   INFO:root:Cluster fsid: 335b6dac-064c-11ea-8243-48f17fe53909
+   INFO:ceph-daemon:Verifying we can ping mon IP 192.168.1.101...
```

Comment:
`s/ceph-daemon/cephadm`

Reply:
I've made this change: $s/ceph-daemon/cephadm